.github/release-draft-template.yml

```diff
@@ -11,13 +11,17 @@ version-resolver:
       - 'new-feature'
   default: patch
 categories:
-  - title: 'Breaking changes ⚠️'
+  - title: '⚠️ Breaking changes'
     label: 'breaking-change'
+  - title: '🚀 Enhancements'
+    label: 'enhancement'
+  - title: '🐛 Bug Fixes'
+    label: 'bug'
+  - title: '🔒 Security'
+    label: 'security'
   - title: 'New features ⚡️'
     label: 'new-feature'
```

Comment:
```suggestion
```

Reply:
@bidoubiwa, should we consider the `enhancements` label to be the label that makes increase the minor, instead of `new-feature` since a new feature is an enhancement?

Reply:
Yes!  It is more flexible as to what we can add to this category. For example, an enhanced method is not perse a new feature but it is an enhancement. And a new feature is also an enhancement. 
I think having one for both is best :)

Reply:
Thanks! commiting this then!